docs/formats/asciidoc-table.md

```diff
@@ -0,0 +1,381 @@
+## terraform-docs asciidoc table
+
+Generate AsciiDoc tables of inputs and outputs
+
+### Synopsis
+
+Generate AsciiDoc tables of inputs and outputs
+
+```
+terraform-docs asciidoc table [PATH] [flags]
+```
+
+### Options
+
+```
+  -h, --help   help for table
+```
+
+### Options inherited from parent commands
+
+```
+      --header-from string             relative path of a file to read header from (default "main.tf")
+      --indent int                     indention level of AsciiDoc sections [1, 2, 3, 4, 5] (default 2)
+      --no-header                      do not show module header
+      --no-inputs                      do not show inputs
+      --no-outputs                     do not show outputs
+      --no-providers                   do not show providers
+      --no-required                    do not show "Required" column or section
+      --no-requirements                do not show module requirements
+      --no-sensitive                   do not show "Sensitive" column or section
+      --no-sort                        do no sort items
+      --output-values                  inject output values into outputs
+      --output-values-from string      inject output values from file into outputs
+      --sort-by-required               sort items by name and print required ones first
+      --sort-inputs-by-required        [deprecated] use '--sort-by-required' instead
+      --with-aggregate-type-defaults   [deprecated] print default values of aggregate types
+```
+
+### Example
+
+Given the [`examples`](/examples/) module:
+
+```shell
+terraform-docs asciidoc table ./examples/
+```
+
+generates the following output:
+
+    Usage:
+
+    Example of 'foo_bar' module in `foo_bar.tf`.
+
+    - list item 1
+    - list item 2
+
+    Even inline **formatting** in _here_ is possible.  
+    and some [link](https://domain.com/)
+
+    * list item 3
+    * list item 4
+
+    ```hcl
+    module "foo_bar" {
+      source = "github.com/foo/bar"
+
+      id   = "1234567890"
+      name = "baz"
+
+      zones = ["us-east-1", "us-west-1"]
+
+      tags = {
+        Name         = "baz"
+        Created-By   = "first.last@email.com"
+        Date-Created = "20180101"
+      }
+    }
+    ```
+
+    Here is some trailing text after code block,  
+    followed by another line of text.
+
+    | Name | Description     |
+    |------|-----------------|
+    | Foo  | Foo description |
+    | Bar  | Bar description |
+
+    == Requirements
+
+    [cols="a,a",options="header",]
+    |===
+    |Name |Version
+    |terraform |>= 0.12
+    |aws |>= 2.15.0
+    |random |>= 2.2.0
+    |===
+
+    == Providers
+
+    [cols="a,a",options="header",]
+    |===
+    |Name |Version
+    |aws |>= 2.15.0
+    |aws.ident |>= 2.15.0
+    |null |n/a
+    |tls |n/a
+    |===
+
+    == Inputs
+
+    [cols="a,a,a,a,a",options="header",]
+    |===
+    |Name |Description |Type |Default |Required
```

Comment:
I'm not too familiar with asciidoc, is it ok that there's no space immediately after `|`s here? Can we use the following instead (may be better readability IMO)

```
| Name | Description | Type | Default | Required
```